.circleci/config.yml

```diff
@@ -106,17 +106,6 @@ jobs:
                 fi;
             fi;
 
-  codecov:
-    <<: *defaults
-
-    steps:
-      - attach_workspace:
-          at: ~/repo
-
-      - run:
-          name: Submit coverage report to codecov
-          command: npm run submit-codecov
-
```

Comment:
Can we just comment it out instead? Will be easier to reintroduce this step when we're ready 😬

Reply:
Nah, forget it